sleap/gui/dataviews.py

```
@@ -386,10 +386,28 @@ def getSelectedRowItem(self) -> Any:
-    def item_to_data(self, obj, item):
-        return {key: getattr(item, key) for key in self.properties}
+    properties = (
+        "name",
+        "filepath",
+        "frames",
+        "height",
+        "width",
+        "channels",
+    )
+
+    def item_to_data(self, obj, item: "Video"):
```

Ruff (0.8.2)

Undefined name `Video`

(F821)